lib/src/core/better_player.dart

```diff
@@ -60,6 +61,11 @@ class _BetterPlayerState extends State<BetterPlayer>
   ///Subscription for controller events
   StreamSubscription? _controllerEventSubscription;
 
+  // For PIP status handling.
+  final EventChannel _eventChannel =
```

Comment:
```suggestion
  final EventChannel _pipStatusEventChannel =
```
If this event channel is only for pip status, I think we can clarify it.